modules/virt-defining-a-watchdog.adoc

```diff
@@ -0,0 +1,93 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-configuring-a-watchdog.adoc
+
+[id="virt-defining-a-watchdog"]
+= Defining a watchdog device
+
+Define how the watchdog proceeds when the operating system (OS) no longer responds.
+
+.Available actions
+[horizontal]
+`poweroff`:: The virtual machine (VM) powers down immediately.
+`reset`:: The VM reboots in place and the guest OS cannot react. Because the length of time required for the guest OS to reboot can cause liveness probes to timeout, use of this option is discouraged. This timeout can extend the time it takes the VM to reboot if cluster-level protections notice the liveness probe failed and forcibly reschedule it.
+`shutdown`:: The VM gracefully powers down by stopping all services.
+
+
+.Procedure
+
+. Create a YAML file with the following contents:
+
++
+[source,yaml]
+----
+---
```

Comment:
I think you can remove this `---` because most of our YAML file examples start with the `apiVersion` line.